packages/@aws-cdk/aws-s3-assets/lib/asset.ts

```diff
@@ -63,9 +69,9 @@ export class Asset extends cdk.Construct implements assets.IAsset {
 
   /**
    * Attribute which represents the S3 URL of this asset.
-   * @example https://s3.us-west-1.amazonaws.com/bucket/key
+   * @example s3://bucket/key
    */
-  public readonly s3Url: string;
```

Comment:
Isn't this a breaking change?

Reply:
Basically we can't introduce a breaking change, so reintroduce this property and add `@deprecated <MESSAGE>` to it.